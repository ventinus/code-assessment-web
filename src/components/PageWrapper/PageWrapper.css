.page-wrapper {
  padding: 30px 15px;
  max-width: 400px;
  margin: 0 auto; }
  @media (min-width: 768px) {
    .page-wrapper {
      max-width: 700px;
      padding: 40px 50px 50px; } }
  @media (min-width: 768px) {
    .page-wrapper {
      max-width: 1120px;
      padding: 55px 80px 70px; } }
