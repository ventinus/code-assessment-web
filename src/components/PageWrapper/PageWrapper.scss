@import '../../scss/variables/mixins.scss';
@import '../../scss/variables/variables.scss';

.page-wrapper {
  padding: 30px 15px;
  max-width: 400px;
  margin: 0 auto;

  @include breakpoint(md) {
    max-width: 700px;
    padding: 40px 50px 50px;
  }

  @include breakpoint(md) {
    max-width: 1120px;
    padding: 55px 80px 70px;
  }
}
