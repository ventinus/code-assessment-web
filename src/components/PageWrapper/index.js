import React from 'react'
import PropTypes from 'prop-types'

import './PageWrapper.css'

const PageWrapper = ({ children }) => <div className="page-wrapper">{children}</div>

PageWrapper.propTypes = {
  children: PropTypes.any
}

export default PageWrapper

