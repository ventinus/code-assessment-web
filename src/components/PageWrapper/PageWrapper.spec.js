import React from 'react'
import renderer from 'react-test-renderer'
import PageWrapper from '.'

describe('PageWrapper component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<PageWrapper>children</PageWrapper>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
