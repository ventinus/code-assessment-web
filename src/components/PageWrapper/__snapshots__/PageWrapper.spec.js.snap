// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageWrapper component renders correctly 1`] = `
<div
  className="page-wrapper"
>
  children
</div>
`;
